app/views/cloud_network/new.html.haml

```diff
@@ -1,51 +1,66 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "cloudNetworkFormController"}
+%form#form_div{:name           => "angularForm",
+   "form-changed"  => true,
+   "miq-form"      => true,
+   "model"         => "vm.cloudNetworkModel",
+   "model-copy"    => "vm.modelCopy",
+   "ng-cloak"      => "",
+   "ng-controller" => "cloudNetworkFormController as vm",
+   "ng-show"       => "vm.afterGet"}
+
   = render :partial => "layouts/flash_msg"
+
   %h3
-    = _('Network Management Provider')
+    = _('Network Provider')
   .form-horizontal
     .form-group{"ng-class" => "{'has-error': angularForm.ems_id.$invalid}"}
       %label.col-md-2.control-label
         = _('Network Manager')
       .col-md-8
         = select_tag("ems_id",
           options_for_select([["<#{_('Choose')}>", nil]] + @network_ems_provider_choices.sort),
-                      "ng-model"                    => "cloudNetworkModel.ems_id",
+                      "ng-change"                   => "vm.filterNetworkManagerChanged(vm.cloudNetworkModel.ems_id)",
+                      "ng-model"                    => "vm.cloudNetworkModel.ems_id",
+                      "pf-select"                   => true,
                       "required"                    => "",
-                      :miqrequired                  => true,
-                      :checkchange                  => true,
-                      "ng-change"                   => "filterNetworkManagerChanged(cloudNetworkModel.ems_id)",
                       "selectpicker-for-select-tag" => "")
-  = render :partial => "common_new_edit"
-  %h3
-    = _('Network Provider Information')
-  .form-horizontal
-    .form-group{"ng-class" => "{'has-error': angularForm.provider_network_type.$invalid}"}
-      %label.col-md-2.control-label
-        = _('Provider Network Type')
-      .col-md-8
-        = select_tag("provider_network_type",
-          options_for_select([["<#{_('Choose')}>", nil]] + @network_provider_network_type_choices.sort),
-                      "ng-model"                    => "cloudNetworkModel.provider_network_type",
-                      :checkchange                  => true,
-                      "selectpicker-for-select-tag" => "")
-  %h3
-    = _('Placement')
+        %span.help-block{"ng-show" => "angularForm.ems_id.$error.required"}
+          = _("Required")
   .form-horizontal
```

Comment:
@gildub Slight adjustment needed to align the `Placement` label with the `Network Provider` label
```diff
--- a/app/views/cloud_network/new.html.haml
+++ b/app/views/cloud_network/new.html.haml
@@ -25,11 +25,10 @@
                       "selectpicker-for-select-tag" => "")
         %span.help-block{"ng-show" => "angularForm.ems_id.$error.required"}
           = _("Required")
-  .form-horizontal
-    .form-group{"ng-class" => "{'has-error': angularForm.cloud_tenant_id.$invalid}",
-                "ng-if"    => "vm.cloudNetworkModel.ems_id"}
-      %h3
-        = _('Placement')
+  .form-horizontal{"ng-class" => "{'has-error': angularForm.cloud_tenant_id.$invalid}", "ng-if" => "vm.cloudNetworkModel.ems_id"}
+    %h3
+      = _('Placement')
+    .form-group
       %label.col-md-2.control-label
         = _('Cloud Tenant')
       .col-md-8
```